### PR TITLE
New Feature

### PR DESCRIPTION
**Pull request by AzamGameChanger**

### CODE REVIEW

File: src/com/bankapp/Users/Customer.java [hunk 1]
```diff
@@ -20,16 +19,28 @@ public Customer (int ID, String name, String phoneNumber,BankAccount checkingAcc
...
-   }
+        throw new IllegalArgumentException("Not found bank account : " + accountID);
+    }
+
+    public BankAccount getAccountType(String type){
+//        ArrayList<BankAccount>types=new ArrayList<>();
+
+        for(BankAccount accountType: accounts) {
+            if (accountType.getType().equals(type)) {
+                accounts.add(accountType);
```
Comment (AminPlusPlus) [inline src/com/bankapp/Users/Customer.java:38]: why are you using "accounts" ?

Comment (AzamGameChanger) [reply src/com/bankapp/Users/Customer.java:38]: > why are you using "accounts" ?

I getting Bank Account types from my accounts. What should you do instead? Creating another Array List doesn't make sense I think.

Comment (AminPlusPlus) [reply src/com/bankapp/Users/Customer.java:38]: I believe inside the method you need an empty ArrayList to add a specific type of account then return

**Review by AminPlusPlus — CHANGES_REQUESTED** [review]
push the last changes

**Pull request merged**